app/models/parent_object.rb

```diff
@@ -78,8 +79,11 @@ def self.extent_of_digitizations
     [nil, "Completely digitized", "Partially digitized"]
   end
 
-  validates :visibility, inclusion: { in: visibilities, allow_nil: true,
-                                      message: "%{value} is not a valid value" }
+  def validate_visibility
+    return true if ['Private', 'Public', 'Redirect', 'Yale Community Only'].include?(visibility)
```

Comment:
Can we reference `self.visibilities` here instead of repeating the constants?

Reply:
I don't think I can call a self.method in a before_save callback because the object doesn't exist yet so it doesn't know what self is. I can change `def self.visibilities` to `def visibilities` and it works, but it breaks the dropdown in the Create PO form. It looks like `self.visibilities` is only used for that form dropdown so I can change the method to the normal `def visibilities` and add the array directly to the `form.select` dropdown if we want?

Reply:
Actually, i tlooks like `self.visibilities` is being used in updatatble as well.

Reply:
I meant access it as a class-level method via `Parent.visibilities`